openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,15 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
+        scope=Scope.content,
+        help="Do not use. For backwards compatibility only."
```

Comment:
This option is hidden and the new option looks like this: 
![screen shot 2015-07-06 at 11 20 16 am](https://cloud.githubusercontent.com/assets/465548/8529703/1c28bf8e-23d1-11e5-8c20-619b670a946e.png)
The old option is kept there for backward compatibility.